lace/Cargo.toml

```diff
@@ -31,14 +31,14 @@ name = "lace"
 path = "src/lib.rs"
 
 [dependencies]
-lace_cc = { path = "lace_cc", version = "0.5.0" }
-lace_utils = { path = "lace_utils", version = "0.2.0" }
-lace_stats = { path = "lace_stats", version = "0.2.1" }
-lace_codebook = { path = "lace_codebook", version = "0.5.0", default_features=false}
-lace_geweke = { path = "lace_geweke", version = "0.2.1" }
+lace_cc = { path = "lace_cc", version = "0.6.0" }
+lace_utils = { path = "lace_utils", version = "0.3.0" }
+lace_stats = { path = "lace_stats", version = "0.3.0" }
+lace_codebook = { path = "lace_codebook", version = "0.6.0", default_features=false}
+lace_geweke = { path = "lace_geweke", version = "0.3.0" }
```

Comment:
Note: I didn't change the version of `lace_consts` since the only change in it was a PATCH version bump in the `rv` dependency. I _think_ that shouldn't require publishing a new version since Cargo should do the right thing and update the patch by default when people build it.